pkg/parsers/azure/azure.go

```diff
@@ -42,6 +42,14 @@ func (g *AzureParser) Parse(azurePipeline *azureModels.Pipeline) (*models.Pipeli
 		pipeline.Jobs[0].Steps = parseSteps(azurePipeline.Steps)
 	}
 
+	if azurePipeline.Pool != nil {
+		pipeline.Jobs[0].Runner = parsePool(azurePipeline.Pool, pipeline.Jobs[0].Runner)
+	}
+
+	if azurePipeline.Container != nil {
+		pipeline.Jobs[0].Runner = parseContainer(azurePipeline.Container, pipeline.Jobs[0].Runner)
+	}
+
```

Comment:
Is it a wanted behaviour if we already have jobs and not only a default job?

Reply:
I don't think it is possible, but you're right I'll change that